docs/start/getting-started/fragments/flutter/generate-model.md

```diff
@@ -0,0 +1,27 @@
+With the basic setup complete, you can now model the data your application will work with. Amplify DataStore persists the modeled data on your local device and can even synchronize to a backend API without writing any additional code. These models are specified as [GraphQL](http://graphql.org/) schemas. You can [learn more](~/cli/graphql-transformer/overview.md)) about GraphQL schemas and data modeling. For now, let’s start by generating some initial data models.
+
+<amplify-callout>
+
+**Did you know?** You can get started with Amplify’s [admin UI](https://sandbox.amplifyapp.com/) without even creating an AWS Account.
```

Comment:
This is an interesting selling point of the Admin UI and could potentially remove a lot of friction from the tutorial for users just looking for a quick try out. However, in the first page "Create an AWS account" is defined as a pre-requisite.

@renebrandel thoughts on this?

It sounds like we should rethink what the pre requisites for the Tutorial actually are. Maybe defer the AWS account creation until it's time to Sync data to the cloud?

Reply:
nit: Admin UI instead of admin UI?